src/chemistry_functionality.jl

```diff
@@ -346,3 +346,16 @@ function create_matrix(reaction::Catalyst.Reaction)
 
     return A
 end
+
+"""
+    balance_system(rs::ReactionSystem)
+
+From a system, creates a new system where each reaction is a balanced version of the corresponding
+reaction of the original system. For more information, consider the `balance_reaction` function
+(which is internally applied to each system reaction).
+"""
+function balance_system(rs::ReactionSystem)
```

Comment:
What does this do if there are no solutions / infinitely many solutions?

Reply:
It seems like it ignores those cases?

Reply:
I think we should throw an error, right?

Reply:
If you get back multiple basis functions you should at least give a warning and pick one. If you get one you should give an error that it can’t balanced.